OWNERS

```diff
@@ -2,8 +2,8 @@
 # Approvers listed here is just a backup in case someone is out.
 approvers:
   - andreyvelich
-  - james-jwu
-  - jbottum
```

Comment:
@franciscojavierarceo As we described here, please can we move the existing approvers to the `emeritus_approvers` section in the OWNERS files ?

https://github.com/kubeflow/community/blob/413f26c6c8a7ec3a1c3db19134323d1c5b340700/KUBEFLOW-STEERING-COMMITTEE.md#:~:text=Update%20approvers%20for%20the%20following%20OWNERS%20files%20(e.g%20the%20past%20members%20should%20be%20moved%20to%20emeritus_approvers)%3A

Reply:
Updated in a follow up PR!